tests/actions/ReportTest.js

```diff
@@ -13,7 +13,30 @@ PushNotification.register = () => {};
 PushNotification.deregister = () => {};
 
 describe('actions/Report', () => {
-    it('should store a new report action in Onyx when one is handled via Pusher', () => {
+    beforeAll(() => {
```

Comment:
Why did we not have this in `beforeAll` before this PR and need it now? Not saying you're wrong, I just actually don't know :)

Reply:
Good question - I believe it's because there was only 1 test in this file before, so there was no need to do `beforeAll`. Now `beforeAll` is useful b/c it sets up Pusher stuff **once**, instead of once for each test